runtime/onert/core/src/backend/builtin/train/KernelGenerator.cc

```diff
@@ -69,7 +69,7 @@ void KernelGenerator::visit(const ir::train::operation::Permute &node)
 
   // NOTE IOTensors of graph outputs for passing data to users must be ignored in training
   //      because the buffers of those IOTensors are unnecessary and nullptr
-  bool ignore_forward_in_training = _tgraph.getOutputs().contains(output_index);
+  bool ignore_forward_in_training = _whole_graph_outputs.contains(output_index);
```

Comment:
@ragmani What is the difference between `_tgraph.getOutputs()` and `_whole_graph_outputs`? I guess `_whole_graph_outputs` > `getOutputs()` from the word `whole`. But I don't understand exactly.

Reply:
As I understand, the `_tgraph` can be a subgraph of whole graph. The Permute op should be ignored when it is an output of whole graph. So he changed the code like this.

Reply:
@jyoungyun Thank you for explanation. But I am curious about what I've asked.

> What is the difference between _tgraph.getOutputs() and _whole_graph_outputs.

Reply:
> Is `partial` related to the following?

Yes, `_tgraph` is a partial graph for `builtin` backend.

> What is the whole and what does it contain more?
It's a whole subgraph.